test/test_browser.py

```diff
@@ -5817,6 +5817,7 @@ def test_audio_worklet(self, args):
     if '-sMEMORY64' in args and is_firefox():
       self.skipTest('https://github.com/emscripten-core/emscripten/issues/19161')
     self.btest_exit('webaudio/audioworklet.c', args=['-sAUDIO_WORKLET', '-sWASM_WORKERS'] + args)
+    self.btest_exit('webaudio/audioworklet.c', args=['-sAUDIO_WORKLET', '-sWASM_WORKERS', '-sSTRICT'] + args)
```

Comment:
Can you just add `-sSTRICT` to the line above?

Reply:
Or if you prefer you could add `strict` key the `@parameterized` dictionary above.